lib/platform/git/private-key.ts

```diff
@@ -0,0 +1,45 @@
+import os from 'os';
+import path from 'path';
+import fs from 'fs-extra';
+import { PLATFORM_GPG_FAILED } from '../../constants/error-messages';
+import { logger } from '../../logger';
+import { exec } from '../../util/exec';
+
+let gitPrivateKey: string;
+let keyId: string;
+
+export function setPrivateKey(key: string): void {
+  gitPrivateKey = key;
+}
+
+async function getKeyId(): Promise<void> {
```

Comment:
I think this should be renamed to
```suggestion
async function importKey(): Promise<void> {
```